pyaedt/modules/MaterialLib.py

```diff
@@ -463,15 +463,23 @@ def add_material_sweep(self, swargs, matname):
         return index
 
     @pyaedt_function_handler()
-    def duplicate_material(self, material, new_name):
+    def duplicate_material(self, material_name, new_name=None, props=None):
         """Duplicate a material.
 
         Parameters
         ----------
-        material : str
+        material_name : str
             Name of the material.
         new_name : str
             Name for the copy of the material.
+        props : list
```

Comment:
@dcrawforAtAnsys should be optional and set to None.

Reply:
docstring was updated.